script/generateUtilsDoc.sh

```diff
@@ -14,8 +14,26 @@
 # You should have received a copy of the GNU General Public License along
 # with this program; if not, see <http://www.gnu.org/licenses/>.
 
+NEW_DEPLOY_NEEDED=0
+
+if [ -z $GITHUB_TOKEN  ]
+then
+    NEW_DEPLOY_NEEDED=0
+    exit 0
+fi
+
 cd $TRAVIS_BUILD_DIR
 
 touch docs/utils.html
 pod2html --infile=lib/utils.pm --outfile=docs/utils.html
 
+#checkout old docs and compare to new ones, then toggle flag accordingly
+git diff gh-pages -- docs/utils.html
+retVal=$?
+if [ retVal -eq 0 ]
```

Comment:
this here needs to be ${retVal} also please use all lower case or snake_case. The condition needs to be -ne, as from the manual:

**Make the program exit with codes similar to diff(1). That is, it exits with 1 if there were differences and 0 means no differences.**

Reply:
@DrMullings can you also change the casing to snake_case please? It's picky I know, but makes it look better overall

Reply:
I adapted the casing and the return values, lets talk about the diff mechanics when you're back